style.css

```diff
@@ -329,3 +329,109 @@ button:hover {
   font-size: 1.3em;
   margin: 0;
 }
+
+#container2 {
+  border-top-left-radius: 105px;
+  display: flex;
+  flex-direction: column;
+  padding: 0 30px;
+  background: #6070ff url(images/contact-form-shapes-mobile.png) no-repeat center;
+  background-size: 100% 90%;
+  margin-top: 20%;
+}
+
+#container2 h1 {
+  font-family: 'Poppins', sans-serif;
+  width: 100%;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2.2em;
+  line-height: 52px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  margin-top: 50%;
+}
+
+#container2 p {
+  font-family: 'Poppins', sans-serif;
+  width: 100%;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1em;
+  line-height: 28px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  color: #ebebff;
+  margin: 0 auto;
```

Comment:
According to the program requirement, you must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) .Please use the font size given in the figma design to allow for consistency with your project 😄